src/components/VeggyHeader/HeaderCart/HeaderCart.jsx

```diff
@@ -21,9 +30,9 @@ export default function HeaderCart (props) {
                     <strong> {getAllProductSum}</strong>
                 </li>
             </ul>
-            <div className={props.shakingCart ? 'cart active' : 'cart'}
+            <div className={shakingCart ? 'cart active' : 'cart'}
```

Comment:
У тебе вже були подібні зауваження. Глянь уважніше: у тебе дублюється класс cart в обох умовах. 

Перепиши 
```suggestion
            <div className={`cart ${shakingCart ? "active" : ""}`}
```